packages/cerebral-provider-firebase/src/deleteUser.js

```diff
@@ -1,9 +1,13 @@
+import {noop as noReturnValue} from './helpers'
 import firebase from 'firebase'
+import {FirebaseProviderError} from './errors'
 
 export default function deleteUser (password) {
-  return new Promise((resolve, reject) => {
-    const user = firebase.auth().currentUser
+  const user = firebase.auth().currentUser
 
-    return user.delete().then(() => resolve(), (error) => reject({error}))
-  })
+  return user.delete()
+    .then(noReturnValue)
+    .catch((error) => {
+      throw new FirebaseProviderError(error.message)
```

Comment:
@fmal Great catch! It should be error... kinda weird, I remember explicitly changing this to use error, going through the actions... hm... must have missed something. Anyways, great! :)